pkg/network/vmicontroller/status.go

```diff
@@ -34,15 +34,49 @@ import (
 )
 
 func UpdateStatus(vmi *v1.VirtualMachineInstance, pod *k8scorev1.Pod) error {
-	interfaceStatuses, err := calculateSecondaryIfaceStatuses(vmi, multus.NetworkStatusesFromPod(pod))
+	var interfaceStatuses []v1.VirtualMachineInstanceNetworkInterface
+
+	networkStatuses := multus.NetworkStatusesFromPod(pod)
```

Comment:
To me this raises a question to me of whether assertion should continue if the length of `networkStatuses` is `<= 1`.

Reply:
`multus.NetworkStatusesFromPod(pod)` can return an empty slice to a slice with N elements.
The functions which calculate the interfaces statuses take it into account.